docker/init.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE graphite;
+
+CREATE TABLE graphite.metrics (
+    date Date DEFAULT toDate(0),
+    name String,
+    level UInt16,
+    parent String,
+    updated DateTime DEFAULT now(),
+    status Enum8('SIMPLE' = 0, 'BAN' = 1, 'APPROVED' = 2, 'HIDDEN' = 3, 'AUTO_HIDDEN' = 4)
+) ENGINE = ReplacingMergeTree(date, (parent, name), 1024, updated);
```

Comment:
Let's use modern engine definition from CH [docs](https://clickhouse.yandex/docs/en/operations/table_engines/replacingmergetree/#creating-a-table): 

```
ENGINE = ReplacingMergeTree(updated)
PARTITION BY toYYYYMM(date)
ORDER BY (parent, name)
SETTINGS index_granularity = 1024;
```

Reply:
+1

Reply:
I copied it from `/doc/install.md`, should I change it there as well?

Reply:
#117

Reply:
updated